pom.xml

```diff
@@ -748,6 +754,9 @@
                         <!-- CWE-611: Improper Restriction of XML External Entity Reference ('XXE') (9.8); -->
                         <!-- Used in tests only. -->
                         <exclude>CVE-2020-36641</exclude>
+                        <!-- Vulnerabilities in io.netty:netty-handler:jar:4.1.99.Final -->
+                        <!-- CWE-300: Channel Accessible by Non-Endpoint ('Man-in-the-Middle') (6.5); -->
```

Comment:
Add a note which dependency requires `netty-handler` and add a justification why it's ok to exclude it.

Reply:
It is really hard to tell who depends on this netty-handler, as it was pinned already.
Justification is simple - no recent version which fixes this CVE, not that much we can do.